backend/src/LibExecutionStdLib/LibNoModule.fs

```diff
@@ -79,6 +79,33 @@ let rec equals (a : Dval) (b : Dval) : bool =
   | DError _, _
   | DIncomplete _, _ -> Exception.raiseCode "Both values must be the same type"
 
+and equalsDType (actual : DType) (expected : DType) : bool =
```

Comment:
Isn't this just `actual = expected`?

Reply:
Yeah. I was trying (maybe in vain) to catch any future case in which DType might have an `id` component to it, to make sure comparisons wouldn't consider the ID. But, I doubt it ever will, so this can be simplified.